docs/pages/modules/module-api.mdx

```diff
@@ -483,6 +486,102 @@ The implementation for three Either types is currently provided out of the box,
 
 </APIBox>
 
+## Guides
+
+<APIBox header="Sending events">
+
+While the JavaScript -> Native communication is mostly covered by native functions, you might also want to let the JavaScript code know about some certain system events,
+for example when the clipboard content changes.
```

Comment:
```suggestion
While JavaScript to Native communication is mostly covered by native functions, you might also want to let the JavaScript code know about certain system events, for example, when the clipboard content changes.
```

Updaing some language + removing the symbol `->` as per our writing styleguide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#symbols-as-words (the styleguide may only talk about & and + but I think in this case it is valid for "->" too).